libnmstate/ifaces/ovs.py

```diff
@@ -190,11 +192,49 @@ def parent(self, value):
     def need_parent(self):
         return True
 
+    @property
+    def patch_config(self):
+        return self._info.get(OVSInterface.PATCH_CONFIG_SUBTREE)
+
     def state_for_verify(self):
         state = super().state_for_verify()
         _convert_external_ids_values_to_string(state)
         return state
 
+    @property
+    def is_patch_port(self):
+        return self.patch_config and self.patch_config.get(
+            OVSInterface.Patch.PEER
+        )
+
+    @property
+    def peer(self):
+        return self.patch_config.get(OVSInterface.Patch.PEER, None)
```

Comment:
by default the `get()` will return None if not found.

Reply:
Done, thanks!